cmd/snap/cmd_snap_op.go

```diff
@@ -45,7 +45,8 @@ var (
 )
 
 var longInstallHelp = i18n.G(`
-The install command installs the named snaps in the system.
+The install command installs the named snaps in the system under a given
+instance name.
```

Comment:
paging @degville

Reply:
I think I'd make the first line simple, ie. `The install command installs the named snaps on the system.`

And add something like the following on a second line:
`To install multiple instances of the same snap, append an underscore and a unique identifier (for each instance) to a snap's name.`